docs/specs/forecast-api.md

```diff
@@ -277,7 +257,13 @@ format:
   "features": [
       {
         "type": "Feature",
-        "properties": {},
+        "properties": {
+            "aircraft_class": "default",
+            "timestamp": "<%Y-%m-%dT%H:%M:%S>",
```

Comment:
probably makes sense to suffix `Z` here  for avoidance of doubt (`"<%Y-%m-%dT%H:%M:%SZ>"`), if there is a future possibility of accepting timezone aware strings in the `time` query param.

Reply:
this will also be renamed to `time`